lib/mix/lib/mix/tasks/deps.clean.ex

```diff
@@ -10,7 +10,7 @@ defmodule Mix.Tasks.Deps.Clean do
   Since this is a destructive action, cleaning of dependencies
   can only happen by passing arguments/options:
 
-    * `dep1, dep2` - the name of dependencies to be deleted
+    * `dep1 dep2`  - the name of dependencies to be deleted separated by a space
```

Comment:
FWIW, I think this is true for all aligned lists in documentation; Markdown ignores these spaces anyways, so we gain very very little by having these aligned.